raiden-dapp/tsconfig.json

```diff
@@ -32,6 +32,9 @@
       "rxjs/*": [
         "node_modules/rxjs/*"
       ],
+      "ethers": [
+        "node_modules/ethers/"
+      ],
```

Comment:
@andrevmatos I spend a day trying to figure out why the build would fail. That is the reason.